scripts/change_headers.swift

```diff
@@ -0,0 +1,122 @@
+#!/usr/bin/swift
+/*
+ * Copyright 2020 LLC Google
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+// Utility script for updating to repo-relative headers.
+
```

Comment:
Great! 
nit: A README or a simple comment explaining how to run the script and what expected result it would be really helpful.

Reply:
Yep. It's still WIP as I work through a first pass through the repo. It's not ready for others to use - but I wanted to check it in to associate with the PR.

The script gets further refined in #5824 and I plan to make it more robust and fully document it before finishing a pass through the repo with it.